src/LeanCode.ContractsGenerator/Program.cs

```diff
@@ -18,10 +19,8 @@ public class ProjectOptions : IOptions
     {
         public string OutputFile { get; set; }
 
-        [Option('s', "solution", Required = false, MetaValue = "FILE", HelpText = "The solution that the project is in.")]
-        public string? SolutionFile { get; set; }
-        [Option('p', "project", Required = true, MetaValue = "FILE", HelpText = "The project file with contracts.")]
-        public string ProjectFile { get; set; }
+        [Option('p', "project", Required = true, Separator = ';', MetaValue = "FILE", HelpText = "The project file with contracts. To pass multiple projects, separate the paths with ';'.")]
```

Comment:
Using `;` as separator will make this awkward in POSIX shells, can use something like `,` instead? Or preferably just consume all following args as they were separated by the OS?

Reply:
Considering how the binary is invoked, there is no benefit in using other character.

Reply:
And if it doesn't matter, why not choose something that's guaranteed not to cause issues instead of something that expected not to cause issues with the exact one way our clients invoke the generator? There's no need to make this harder for potential future clients.

Reply:
> Also, unfortunately, because of the `CommandLine` library limitations, this is the only way to handle these.

Then maybe we should have used a different library? :upside_down_face:

Reply:
> Then maybe we should have used a different library? 🙃
That is totally unnecessary comment that brings no benefit to the conversation. Really, making passive-aggresive comments brings nothing to the conversation.

Reply:
>And if it doesn't matter, why not choose something that's guaranteed not to cause issues instead of something that's just expected not to cause any with the exact one way our clients invoke the generator? There's no need to make this harder for potential future clients, or make manual invokation for testing purposes harder than it already is.

I don't have means to change that now. Will change that the next time I have access to my dev machine.

Reply:
> That is totally unnecessary comment that brings no benefit to the conversation. Making passive-aggresive comments brings nothing to the conversation, really.

Eh, it was intended to poke fun at the sillyness of the situation in which a choice of command line parsing library we have made 1.5 months ago already became technical debt of sorts, sorry. I have a feeling this wouldn't be a problem if we just globbed on `string[] args` and handled the result based on extension.